server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "body-parser": "^1.18.3",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v2.4.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#td-multiple-fallbacks",
```

Comment:
Remember to update this once https://github.com/bravetechnologycoop/brave-alert-lib/pull/10 is merged.